lib/kite.rb

```diff
@@ -10,5 +10,6 @@
 require 'kite/base'
 require 'kite/render'
 require 'kite/generate'
+require 'kite/user_tasks_autoload'
```

Comment:
I wanted to have kite/cloud.rb load user tasks only if cloud.yml was successfully parsed
or find a way to do lazy loading